app/workers/base_worker.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class BaseWorker
```

Comment:
To follow RoR naming convention propose to rename it to `ApplicationWorker`

Also, this worker should not have any implementation, for sentry we need to have a separate worker which will be inherited from the `ApplicationWorker`